From 150191ae2359732b4c15d220c854d010fcc88e39 Mon Sep 17 00:00:00 2001
From: Peter Ujfalusi <peter.ujfalusi@ti.com>
Date: Thu, 3 Jul 2014 07:51:56 +0300
Subject: [PATCH 0807/1587] dma: edma: Declare DMA_SLAVE_BUSWIDTH_3_BYTES as
 supported buswidth

commit e4a899d9bd5d18f5568be88f7ec8edf4cd107a94 upstream

eDMA can be configured for 3bytes word size for source and destination.

Signed-off-by: Peter Ujfalusi <peter.ujfalusi@ti.com>
Acked-by: Vinod Koul <vinod.koul@intel.com>
Acked-by: Takashi Iwai <tiwai@suse.de>
Signed-off-by: Mark Brown <broonie@linaro.org>
---
 drivers/dma/edma.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/dma/edma.c b/drivers/dma/edma.c
index ec9c410..72ef2d0 100644
--- a/drivers/dma/edma.c
+++ b/drivers/dma/edma.c
@@ -981,6 +981,7 @@ static void __init edma_chan_init(struct edma_cc *ecc,
 
 #define EDMA_DMA_BUSWIDTHS	(BIT(DMA_SLAVE_BUSWIDTH_1_BYTE) | \
 				 BIT(DMA_SLAVE_BUSWIDTH_2_BYTES) | \
+				 BIT(DMA_SLAVE_BUSWIDTH_3_BYTES) | \
 				 BIT(DMA_SLAVE_BUSWIDTH_4_BYTES))
 
 static int edma_dma_device_slave_caps(struct dma_chan *dchan,
-- 
1.7.5.4

